### PR TITLE
Dashboard: Allow removing min refresh interval from refresh options (5s or other)

### LINKED ISSUE #25842: Can't remove 5s database auto-refresh interval
**What happened**:
The 5s auto-refresh interval is still available for the dashboard after removing it from the dashboard configuration. All other default intervals can be removed.

This is happening on the 7.0.3 version. The version 6.7.4 works fine. It's been reproduced on play.grafana.org as well.

**What you expected to happen**:
That the 5S auto-refresh interval is no longer available after being deleted.

**How to reproduce it (as minimally and precisely as possible)**:
1. Open the dashboard settings screen
2. Remove the 5s interval from the Auto-refresh box in the General settings tab
3. Save the dashboard
4. Reload the dashboard and click on the auto-refresh interval picker. 5s is still available there.

**Anything else we need to know?**:
N/A

**Environment**:
- Grafana version: 7.0.3
- Data source type & version: N/A
- OS Grafana is installed on: CentOS7
- User OS & Browser: N/A
- Grafana plugins: N/A
- Others: N/A

### PR DESCRIPTION
**Pull request by torkelo**
Fixes #25842

### CODE REVIEW

File: public/app/features/dashboard/services/TimeSrv.ts [hunk 1]
```diff
@@ -79,12 +79,7 @@ export class TimeSrv {
       return intervals;
     }
 
-    const validIntervals = intervals.filter(str => str !== '').filter(this.contextSrv.isAllowedInterval);
-
-    if (validIntervals.indexOf(this.contextSrv.minRefreshInterval) === -1) {
-      validIntervals.unshift(this.contextSrv.minRefreshInterval);
```
Comment (torkelo) [inline public/app/features/dashboard/services/TimeSrv.ts:85]: not sure why this logic was added here, @marefr do you know?

**Review by dprokop — APPROVED** [review]
This looks OK

File: public/app/features/dashboard/services/TimeSrv.ts (hunk 1 above)
Comment (marefr) [reply public/app/features/dashboard/services/TimeSrv.ts:85]: No, not sure. If I understand it correctly this adds the min interval to valid intervals if it doesn't exists? Yeah that sounds wrong to me.

**Review by marefr — APPROVED** [review]
LGTM

**Pull request merged**